src/controllers/drivers.cr

```diff
@@ -26,22 +26,19 @@ module PlaceOS::Core::Api
       driver = URI.decode(params["id"])
       commit = params["commit"]
       repository = params["repository"]? || "drivers"
```

Comment:
Given we're making repositories completely DB entry dependent, should make `repository` a requirement for these APIs?